apps/web/src/actions/users/fetch.test.ts

```diff
@@ -1,12 +1,16 @@
-import { getSession } from '$/services/auth/getSession'
-import { createWorkspace } from '$core/tests/factories/workspaces'
```

Comment:
Please review your imports. This was not using the factory but the real service. Always import from `@latitude-data/core`

Reply:
it was using the factory no?